IS0402Test.py

```diff
@@ -2196,24 +2063,32 @@ def post_subscription(self, test, sub_json, query_url=None):
         except json.decoder.JSONDecodeError:
             raise NMOSTestException(test.FAIL("Non-JSON response returned for Query API subscription request"))
 
-    def post_resource(self, test, type, data, code=None):
+    def post_resource(self, test, type, data=None, reg_url=None, codes=[200, 201]):
```

Comment:
I'm not sure it causes a problem in this specific case, but generally using lists/dicts as default arguments can cause some unexpected behaviour: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Sorry. C++ programmer. Fixed!